gatorgrouper/utils/parse_arguments.py

```diff
@@ -33,19 +33,11 @@ def parse_arguments(args):
     )
 
     gg_parser.add_argument(
-        "--group-size",
-        help="Number of students in a group",
-        type=int,
-        default=constants.DEFAULT_GRPSIZE,
-        required=False,
+        "--group-size", help="Number of students in a group", type=int, required=False
     )
 
     gg_parser.add_argument(
-        "--num-group",
-        help="Number of groups",
-        type=int,
-        default=constants.DEFAULT_NUMGRP,
-        required=False,
+        "--num-group", help="Number of groups", type=int, required=False
```

Comment:
Is that necessary to remove the default `numgroup` and `groupsize`? I thought those are helpful though

Reply:
Default values were removed as a result of the changes required in the logic of the if statements in gatorgrouper_cli. If we can find a way to have defaults in, but also check for explicitly set values that would be ideal, but I don't know of a good non-intrusive way to do that.